
*Summary*

More review information. I like what is being said here but need to put my own spin on it - so it can make sense in the face of my own project... which I haven't yet decided on. I think that at a macroscopic level, it'd still be neat to find out what give biofilms their biogeography.

This leaves me with *a lot* to do tomorrow.

(A) Write up a brief summary of measurable factors. I'll need to pull from the reviews and from my *imagination*. This'll be useful for statistical analyses, and for interpretation, both of my work and of others working on macroscopic biofilms.

(B) Chat to someone about taking growth curves for fluorescence (probs Sam. I'm sorry Sam.) I'm not sure what media was to be used and a couple of strain specifics.

(C) Figure out whether some strains are better for polymicrobial assays. This would be papers for Pa/Sa coculture, maybe just a summary table.

(D) Clean up the review that I was currently reading.

(E?) I still have that review on ecological synchrony, and it should be read at some point in the future! But maybe another day.

---

*[Review](https://pubmed.ncbi.nlm.nih.gov/39979272/) summary continued*

*Agar biofilm colonies*

They give some reviews on agar colony biofilms. The borders of the colony, at the air-agar interface, expand both horizontally/vertically. The rate of growth after the exponential-phase is linear. The growth rate vertically, is bound by substrate availability. The growth rate radially, is bound by agar surface tension/friction. The sectors are heterogenous, and they favour faster exponential growth.

Not used for biofilm-biofilm interactions, typically. For this they give only one reference.

Can sample for lineage prevalence, by counting desirable CFU's, but this misses out on space-dependent phenotypes. Some cells may grow better only in the presence of others - metabolic crossfeeding. Not an ideal model for single-cell modelling, but this can be done at the colony's borders. Studies have found success with finding links between community factors: degrees of intermixing, cell morphology, physical interactions, cell-to-cell killing, and expansion lag-time, with the size and shape of colony sectors. Lots of references here. The cutting-edge stuff is *in-situ* sequencing.

*Microfluidic flow-cells*

Can be either 'open', where there is an inflow of new nutrients, and waste is continuously removed, or can be 'closed', where only spent culture is allowed to circulate.

Time-lapse microscopy, be it light or confocal, can follow through biofilms from the phases of attachment to dissipation. It is possible, albeit difficult, to view individual cells.

The flow-cell environment is unusual. The cells only have one source of both nutrients/oxygen. An agar plate colony, as written previously, has these coming from opposite sides. The main forces are also different: friction is the main force acting on agar-plate colonies, where shear force is the main force acting on flow-cell colonies.

There is room for biofilms to meaningfully interact. Upstream colonies, can exchange both cells and nutrients, with those downstream which may specialize otherwise.

*Microfluidic growth-chambers*

These are flow-chambers with one or more secondary branches, that can shelter cells from the shear forces of the primary flow-cell. The chambers are spatially constrained, and are good for looking at biofilms in high-resolution, as would be granted by growth in 2-dimensional space.

The space limitation however, promotes growth of quickly-growing lineages.

*Agar microcolonies*

Agar with a glass slide overlay, best suited towards high-resolution analyses, less so those of microcolonies, and more-so that of trans-colony migration. The tight confines of the colonizable space, could even over-represent motility, in terms of cellular importance. Note that chemical gradients, such as those of signalling molecules, may be weaker when needing to diffuse through agar... Cells may also be under unrealistic stress, as they can have to deal with both intracolony/intercolony relationships.

Ultimately, it's a good idea to minimize confounding, at least when choosing an experimental model. Which one would be best to look at the size~shape relationship, for polymicrobial microcolonies, really? If I assume that each of these models has many cohesive characteristics, at all biological levels, that all guide towards a singular phenotype, then it would make sense to choose a model that covers the hypothesis' weaknesses.

It seems self-sabotaging, but I really like the point that the authors are suggesting. Does the model support the worst-case scenario for our research question?

*Building predictive models*

![model-classes](https://github.com/marklemzin/marks-masters/raw/main/pictures/10.3%20model-classes.webp)

*This is a wee bit of a complicated figure. Given that agar-plate colony biofilms, in all their macroscopic glory, aren't great for getting single-cell resolutions, I think that something akin to (B) is best. (C) is unavoidable, as chaotic modelling, while very cutting-edge, is well and beyond my abilities.*

*Mechanistic models*

Better known as pseudo-mathematical models, these form the bulk of the biological literature. They present answers as biological flow-charts: not unlike metabolic diagrams, or as mathematical agent-based-modelling. It is possible, if difficult, to use such models to answer questions about single-cell dynamics, but this needs highly technical equipment. I like to think of this as the molecular perspective. The real heavyweight literature, top-tier journal publications, hits along these lines and much of the featherweight literature, though still great in their own right only aspire to follow this modelling style.

The trickiest parts of this model are getting a high experimental resolution. It's good for metabolite exchange (both into cells and into their shared medium), physical restrictions, or nutrient leakage. But if some parameters, as given by the experimental system, can be ignored (maybe motility, maybe cell morphology), then it may be alright to use a system that cannot resolve molecular confounding.

*Heuristic models*

Best thought of as the integral to mechanistic-modelling. Here the focus is on spatial gradients, rather than concentrations proper. Most values can be simplified, although at a cost to blurring the data by averaging, either by looking at temporal/spatial gradients. The level of abstraction, however, can be fine-tuned to a particular question. For example, some cells may have chemical interactions, but is it always needed to disentangle them? I guess if Pa both starves and kills Pa directly, these are distinct mechanisms. We'd need to have two models to cover both eventualities, then pick the one with the best predictive power.

Here we care most about generalization. What is true for biofilms even between differing experimental, or thought models?

They mention looking at cell-to-cell interactions, using measures of fitness like the Lotka-Volterra model. This is statistical, but it kind of goes to show how these heuristic models, despite their elegance, are best used with statistical models.

*Example hypotheses*

| Question                                                                                        | Comments                                                                                                                                                                                                           |
| ----------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| What are the factors that govern biofilm pattern making?                                        | In particular, whether the factors given earlier are experiment-specific.                                                                                                                                          |
| Are factors interconnected, in any way?                                                         | This is about confounding. At the same level of resolution, many molecules, for example, can be identified. Whether they help cells grow in conjunction, for example, is not yet well-understood.                  |
| Are spatial patterns enough to infer cell-to-cell dynamics, or vice-versa?                      | About setting biological resolutions. I think of this as how complex each resolution is, for example, if there are many key molecular factors, then it makes sense that a cell-level model would be inappropriate. |
| More specifically, is there a way that the shape of cellular microcolonies, can confer fitness? | Maybe a growth advantage. This is kind of an age-old question in whether organisms, be they cells or microcolonies, are more than the sum of their parts?                                                          |
| What is the role of time?                                                                       | I guess it'd be neat to see whether biofilm maturity, or infancy, can have a kind of altered fate, if we treat them at the right time - it's kind of like developmental genetics.                                  |
| What is the role of cell death?                                                                 | *This one.* The rest were obvious, but this is a very challenging question. What if something is taken away from a biofilm? Be it cell or nutrient, how will it respond?                                           |

*Statistical models*

Super low-resolution, but really just the formal side of heuristic models. I've thought that these are here to resolve experimental chaos, in things like where cells are seeded in a particular media. Sector width is linked to sector mixing for example, is one neat conclusion. This is reference 49, and sounds like what I wanted to study!

*Back to the paper grind...*

I'll try out this [review](https://pubmed.ncbi.nlm.nih.gov/36514980/).

It looks at agar plate colony biofilms. They've been used to look at biofilm-biofilm interactions, polymicrobial interactions (that *can't be predicted by monoculture*), find pathways needed for biofilm production, and to find biofilm matrix constituents.
There's also lots of great mathematical parameters. Some quantifications: strain abundance, spatial segregation, biofilm footprint - ?, growth rate (CFU?), thickness, viscoelastic properties, competitive fitness, mutation rates, and biomass.
*And they give references.*

![cool-biofilms](https://github.com/marklemzin/marks-masters/raw/main/pictures/10.3%20cool-biofilms.jpg)

*This picture taken from the review is I guess Daniel's perspective. I was thinking I would work with something simpler at the beginning. But this is also acceptable, given that I can focus on macroscopic features.*

![cool-analyses](https://github.com/marklemzin/marks-masters/raw/main/pictures/10.3%20cool-analyses.jpg)

*Very similar diagram, at least to the one given above in the previous review. (A) is ABM, (B) is statistical, (C) is cell-automata. Note the changes in resolution.*

The maths they say can be put into 3 categories: partial-differential equations (PDE's, sort of like ecological modelling), agent-based modelling (a bottom-up approach), and cellular automata (my beloved, CA, a top-down approach).

PDE's, are computationally inexpensive. They care about averages, rather than cell-level trends. The review gives some links to software such as Matlab's PDE toolbox, FreeFEM, and Python-based FEniCS.

Big thing with ABM's, is that they can focus on cellular characteristics: cell-shape/orientation, division rates, anything really, but especially stochasticity. It feels a lot like statistical physics. The trouble is tracking each individual agent, and their neighbourhood. This is computationally expensive. The review again gives some links to software like CellModeller, Nanoverse, NUFEB, and iDynoMiCs. Maybe there's some fun to be had with chaos theory here? How often do stronger cells win?

Where the ABM above, it cares more about relative neighbourhoods, we can also assign absolute neighbourhoods. These are CA. Points are no longer cells but rather locales. They aren't often appropriate, but their implementation, and computational drain, are both much better than ABM's. I feel that I can start here and expand out to ABM's, if necessary.

Biofilms are not necessarily visible. Many cells are embedded, and may have other characteristics, and can contribute to heterogeneity. They give some software 'BiofilmQ'/'Comstat', that may help with quantification. Some good parameters (given earlier), are Ripley's K function: for heterogeneity, spatial assortment, neighbour/intermixing indices: for segregation, Shannon's diversity index: for species diversity, and fractal dimension: for the extent of dendritic patterns.

The review cares most about short-term spatial heterogeneity, not over the time needed for spatially-dependent evolution, but over the time needed for 'ecological patterns' to emerge. It's a smaller timeframe that wants to see how cells aggregate, and not how communities mature. I feel that this is acceptable, though a touch unambitious.

I like how they call this spatial mixing. Their term cares most about microcolony development, and not individual cells. For example, cells can arrange into dendritic patterns, but be heterogenous otherwise. Kind of like how a Lego brick can be the base of a little toy house - not always the atom.

Spatial segregation, though relatively simple, is enough to model diffusion-distance antagonism. Close-range cell death may happen by theft of resources, or contact-dependent mechanisms (T6SS). Long-range cell death can happen by QS/toxins. Keeping a barrier, such as with dead cells or a noncombatant cell, could be great as a kind of ecological 'no-man's zone'.

Initial inoculum density, an important concept. Low total densities: good for uncompetitive species (if there is contact-dependent antagonism), not good for parasitic species. Why does spatial segregation occur, is this a key question? The review say that it could be due to an increase in how long it takes single cells to expand and form borders - and that this can sort of be shown by having greater inoculum densities. There is not yet any experimental/theoretical validation.

Cells at the edge of the colony, due to having fewer physical boundaries, can better contribute to heterogeneity. There is one confounder here: 'the coffee ring effect'. Cells get stuck at the edge of the colony, as the biofilm dries. What the effect of this is - is uncertain.

*A look at the fluorescent strains*

I've been told to focus on Pa PA14, PAO1, and LESB58. Sa should be USA300 Lac, I guess but I'll verify later.

*Pa*

| No. in Box | Source | Strain    | Engineering         | Cassette | Comments                                                                                                                                                       |
| ---------- | ------ | --------- | ------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| 127        | De     | LESB58    |                     |          |                                                                                                                                                                |
| 177        | De     | PAO1      |                     |          |                                                                                                                                                                |
| 207        | De     | PAO1      |                     |          |                                                                                                                                                                |
| 4          | NZ     | LESB58    |                     |          |                                                                                                                                                                |
| 6          | NZ     | PA14      |                     |          |                                                                                                                                                                |
| 63         | NZ     | LESB58    | pUCP23.mCherry-1    | Gm500    | Seems to be a standard plasmid. One of Lamont's [papers](https://pubmed.ncbi.nlm.nih.gov/12492864/), as an example. It is used to express an antisigma factor. |
| 64         | NZ     | LESB58    | pUCP23.mCherry-2    | Gm500    |                                                                                                                                                                |
| 88         | NZ     | ?         | pMRP0-1gfp          | Gm50     | ? Weird notation. Also missing a strain...                                                                                                                     |
| 131        | NZ     | LESB58    | pUCP23.mCherry-1    | Gm500    |                                                                                                                                                                |
| 132        | NZ     | LESB58    | pUCP23.mCherry-2    | Gm500    |                                                                                                                                                                |
| 134        | NZ     | PAO1-H103 | pUCP23.mCherry-1    | Gm50     | A little bit of [H103](https://pubmed.ncbi.nlm.nih.gov/118160/). Looks like a PAO1 variant, it just has a couple of extra outer-membrane proteins.             |
| 160        | NZ     | PA14      | pUCP23.mCherry-1    | Gm50     |                                                                                                                                                                |
| 238        | NZ     | PAO1      | pUC-mini-P1-mCherry | Gm50     | This is a chromosomal strain. Not a plasmid!                                                                                                                   |
| 317        | NZ     | PAO1-H103 | pUCP.eGFP-1         | Gm50     | 'e' for enhanced                                                                                                                                               |
| 318        | NZ     | PAO1-H103 | pUCP.eGFP-2         | Gm50     | I guess this means pUCP23. I'm not sure if there is a 'pUCP' or 'pUC'.                                                                                         |
| 323        | NZ     | PA14      | pUCP.eGFP-1         | Gm50     |                                                                                                                                                                |
| 324        | NZ     | PA14      | pUCP.eGFP-2         | Gm50     |                                                                                                                                                                |
| 366        | NZ     | PAO1-H103 | pUC-mini-P1-GFP     | Gm50     |                                                                                                                                                                |
| 367        | NZ     | PAO1-H103 | pUC-mini-P1-GFP     | Gm50     |                                                                                                                                                                |
| 369        | NZ     | PAO1-H103 | pUC-mini-P1-YFP     | Gm50     |                                                                                                                                                                |
| 370        | NZ     | PAO1-H103 | pUC-mini-P1-YFP     | Gm50     |                                                                                                                                                                |
| 380        | NZ     | PAO1 (NZ) |                     |          |                                                                                                                                                                |
| 381        | NZ     | PA14 (NZ) |                     |          |                                                                                                                                                                |
| 442        | NZ     | LESB      | pUC.yfp             | Gm500    |                                                                                                                                                                |
| 443        | NZ     | LESB      | pUC.yfp             | Gm500    |                                                                                                                                                                |
| 445        | NZ     | LESB      | pUC.eGFP            | Gm500    |                                                                                                                                                                |
| 446        | NZ     | LESB      | pUC.eGFP            | Gm500    |                                                                                                                                                                |
| 497        | NZ     | PAO1      |                     |          |                                                                                                                                                                |
No maps for either pUCP23, or for pUC-mini. Guess I'll have to trust what is written here until something else resurfaces.


*Sa*

| No. in Box | Source | Strain     | Engineering         | Cassette                            | Comments                                                                                                                                 |
| ---------- | ------ | ---------- | ------------------- | ----------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
| 71         | NZ     | RN4220     | pSFRFPS1-FarRed     | Tmp10                               | 'FP650: 592/650 nm'. Not sure what this means? [Linked paper](https://pmc.ncbi.nlm.nih.gov/articles/PMC5735104/). They say 10ug/mL Trim. |
| 72         | NZ     | RN4220     | pSFRFPS1-DsRed      | Tmp10                               |                                                                                                                                          |
| 73         | NZ     | RN4220     | pSFRFPS1-GFP        | Tmp10                               |                                                                                                                                          |
| 75         | NZ     | LAC USA300 | pKK.FarRed          | Tmp10                               | This fp is a touch mysterious, no? I'm not sure what it is but I guess I'll try a higher wavelength -                                    |
| 76         | NZ     | LAC USA300 | pKK.FarRed          | Tmp10                               | This must be pKK30 or pKK20                                                                                                              |
| 77         | NZ     | RN4220     | pKK.FarRed          | Tmp10                               |                                                                                                                                          |
| 78         | NZ     | RN4220     | pKK.FarRed          | Tmp10                               |                                                                                                                                          |
| 79         | NZ     | RN4220     | pKK.FarRed          | Tmp10                               |                                                                                                                                          |
| 74         | NZ     | LAC USA300 | 'carrying *gfpopt*' | ? Ay carumba. Is this on a plasmid? | [Linked paper](https://pubmed.ncbi.nlm.nih.gov/25884622/). '488/561 nm'                                                                  |
pSFRFPS1: a name given to a plasmid based off of pKK30, with inserts of fluorescent reporters. It isn't clear what the gene reporter is, but it should be something from pKK30. The 'FP650' mentioned, seems to be shorthand for a protein called [eqFP650](https://www.fpbase.org/protein/eqfp650/), as the wavelengths seem to align with its emission spectra. This does not work in *S. aureus*.

![sfrfps1](https://github.com/marklemzin/marks-masters/raw/main/pictures/10.3%20sfrfps1-map.jpg)

*repF, r6ky,*. Promoter control: *sarA*. Under control by [*sigA/sigB*](https://pmc.ncbi.nlm.nih.gov/articles/PMC2258893/), which help with [host-responses/intracellular-persistence](https://pmc.ncbi.nlm.nih.gov/articles/PMC4414502/). The resistance cassette, isn't apparent, but seems to be *dfrA*, which is a [trimethoprim-insensitive dihydrofolate-reductase](https://academic.oup.com/jac/article/76/11/2748/6310424). I guess that *sarA*, maybe is fused with *fp650*, I guess?

This *gfpopt*, I'm not sure about either - it looks like [this](https://pmc.ncbi.nlm.nih.gov/articles/PMC2663184/)? Not sure entirely. They treat it like eGFP. Maybe it's just a codon-optimized variant. Shouldn't behave any differently. Nice to have Sa with GFP, as a possible option.

*Fluorescent proteins*
Copied from the wonderful [www.fpbase.org](https://www.fpbase.org/).

| Name              | Year | Excitation (nm) | Emission (nm) | Link                                                         |
| ----------------- | ---- | --------------- | ------------- | ------------------------------------------------------------ |
| DsRed             | 1999 | 558             | 583           | [DsRed](https://www.fpbase.org/protein/dsred/)               |
| GFP               | 1992 | 395             | 509           | [GFP](https://www.fpbase.org/protein/avgfp/)                 |
| FarRed            | ?    | ~542            | ~670          | [Far-Red collection](https://www.fpbase.org/collection/709/) |
| eGFP ('enhanced') | 1996 | 488             | 507           | [EGFP](https://www.fpbase.org/protein/egfp/)                 |
| YFP ('eYFP')      | 1996 | 513             | 527           | [EYFP](https://www.fpbase.org/protein/eyfp/)                 |
| mCherry           | 2004 | 587             | 610           | [mCherry](https://www.fpbase.org/protein/mcherry/)           |
I think that I'll roll with these fluorescent proteins. Shouldn't be too tricky to use the basic growth-curve protocol. The only thing is blacked-out 96-wells, which we seem to have some of - I'll hoard some in preparation. I need to ask about the phenol-red-free TCM. No idea where it is in the cold room - but Rory'll know.

*Another Sam debate*
I brought up the idea of volume ~ border-length, and he gave me a couple of valuable ideas. The first is that there should be no funny business, at least when thinking about surface area ~ volume. If that is not linear then there are morphological changes. There is a lot of classic lit on doing this in bacterial cells, but not on microcolonies. It's kind of how cells could have a specialized morphology, for example, when under some kind of environmental stress. I wonder if there is something that makes microcolonies, and their cells more specialized.

It reminds me so much of developmental biology. Cues for morphology, maturation, need to be on time and in the right place - like workers at a 9-to-5. What if we mess with scheduling, maybe we can break the system?

*Brief catch-up with Daniel*
Back at it with Daniel. He told me to get to work with the growth curves but I'm missing a wee bit of information. There was something about phenol-red-free TCM, but that shouldn't be unanswerable. The real trouble is finding the right *Sa* and *Pa* strains for these experiments. He wanted to just roll with the usual suspects: PAO1, LESB58, PA14. I didn't catch what the deal with LAC USA300 was. Guess that's the plan for now though - standard strains and some kind of coculture method.

I'll try do some shaking coculture. It'd be a neat way to assess fp independence. Maybe after the monoculture work.

*Writing a great literature review*

Compare and contrast, that's the key! I dislike information dumps. As useful as they may be they are just too time-consuming to digest.

*Making beautiful colonies*

My best success here was with slow-growing LESB58, on a media permitting motility. Slow growth is best to get fine and sharp colony edges. Not widely applicable, yes - but maybe still extremely useful.