*Summary*

Got started reading some of the papers from reviews - I think I have a better picture of Daniel's goal. It's more about creating weird macroscopic colonies, and not so much about pattern-making at a single-cell resolution.

Today I'll try and get the strains streaked out on agar then continue my readings.

---

*Planning the literature review*
- It'd be nice to have a big ol' table of papers on colony biofilms. I'd need a column for mathematical analysis. I'd have a little section on the pro's and con's of each technique.
- Biofilm inoculum.
- Organism(s) used.

*Strains of interest*

| No. in Box | Source | Strain     | Engineering      | Cassette | Comments                                                                                                                                                            |
| ---------- | ------ | ---------- | ---------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| 75         | NZ     | LAC USA300 | pKK.FarRed       | Tmp10    | This fp is a touch mysterious, no? I'm not sure what it is but I guess I'll try a higher wavelength -                                                               |
| 76         | NZ     | LAC USA300 | pKK.FarRed       | Tmp10    | This must be pKK30 or pKK20                                                                                                                                         |
|            |        |            |                  |          |                                                                                                                                                                     |
| 63         | NZ     | LESB58     | pUCP23.mCherry-1 | Gm500    | Seems to be a standard plasmid. One of Iain Lamont's [papers](https://pubmed.ncbi.nlm.nih.gov/12492864/), as an example. It is used to express an antisigma factor. |
| 64         | NZ     | LESB58     | pUCP23.mCherry-2 | Gm500    |                                                                                                                                                                     |
| 131        | NZ     | LESB58     | pUCP23.mCherry-1 | Gm500    |                                                                                                                                                                     |
| 132        | NZ     | LESB58     | pUCP23.mCherry-2 | Gm500    |                                                                                                                                                                     |
| 442        | NZ     | LESB       | pUC.yfp          | Gm500    |                                                                                                                                                                     |
| 443        | NZ     | LESB       | pUC.yfp          | Gm500    |                                                                                                                                                                     |
| 445        | NZ     | LESB       | pUC.eGFP         | Gm500    |                                                                                                                                                                     |
| 446        | NZ     | LESB       | pUC.eGFP         | Gm500    |                                                                                                                                                                     |
|            |        |            |                  |          |                                                                                                                                                                     |
| 318        | NZ     | PAO1-H103  | pUCP.eGFP-2      | Gm50     | I guess this means pUCP23. I'm not sure if there is a 'pUCP' or 'pUC'.                                                                                              |
| 366        | NZ     | PAO1-H103  | pUC-mini-P1-GFP  | Gm50     |                                                                                                                                                                     |
| 367        | NZ     | PAO1-H103  | pUC-mini-P1-GFP  | Gm50     |                                                                                                                                                                     |
| 369        | NZ     | PAO1-H103  | pUC-mini-P1-YFP  | Gm50     |                                                                                                                                                                     |
| 370        | NZ     | PAO1-H103  | pUC-mini-P1-YFP  | Gm50     |                                                                                                                                                                     |
|            |        |            |                  |          |                                                                                                                                                                     |
| 160        | NZ     | PA14       | pUCP23.mCherry-1 | Gm50     |                                                                                                                                                                     |
|            |        |            |                  |          |                                                                                                                                                                     |
| 380        | NZ     | PAO1 (NZ)  |                  |          |                                                                                                                                                                     |
| 381        | NZ     | PA14 (NZ)  |                  |          |                                                                                                                                                                     |
Note that the chromosomal strains, despite having a more stable fp gene - are not better fp producers. This is due to there being only around one bacterial chromosome, and many possible plasmids.

*And one more thing!*

I forgot to write down a growth-curve protocol.

![growth-fl-curve]

*This is roughly the sort of plot I'd be looking for - two lines on a single plot!*

From [this methods](https://pubmed.ncbi.nlm.nih.gov/34260309/) paper: adapted

The strains used are listed in the table above called *Strains of interest*. Pa cells were grown in Luria Broth (LB) routinely, and Sa cells were grown in Tryptic Soy Broth (TSB) routinely, with added 50ug/mL Gentamicin, 500ug/mL Gentamicin, or 10ug/mL Trimethoprim, as appropriate. Agar cultures were grown at 37Â°C.

The plate reader growth curves were carried out in 96-well microtiter plates (GBO). Cultures were put in wells with up to 200 uL per well. For these cultures the bacteria were harvested, from agar plates then resuspended, in 1 X PBS (PBS tablets; Gibco), up to a final optical density (OD600; 600nm) of 0.02. Cells were centrifuged, where appropriate, at 5000rpm for 5 minutes. Cells were shaken at 250 rpm in-between readings. The cycle of measurements, was once every 10 minutes.

The emission/excitation spectra are below.

| Name              | Year | Excitation (nm) | Emission (nm) | Link                                                         | Strain no.                           |
| ----------------- | ---- | --------------- | ------------- | ------------------------------------------------------------ | ------------------------------------ |
| GFP               | 1992 | 395             | 509           | [GFP](https://www.fpbase.org/protein/avgfp/)                 | PAO1: 366, 367.                      |
| FarRed            | ?    | ~542            | ~670          | [Far-Red collection](https://www.fpbase.org/collection/709/) | LAC USA300: 75, 76.                  |
| eGFP ('enhanced') | 1996 | 488             | 507           | [EGFP](https://www.fpbase.org/protein/egfp/)                 | LESB58: 445, 446. PAO1: 318.         |
| YFP ('eYFP')      | 1996 | 513             | 527           | [EYFP](https://www.fpbase.org/protein/eyfp/)                 | LESB: 442, 443. PAO1: 369, 370.      |
| mCherry           | 2004 | 587             | 610           | [mCherry](https://www.fpbase.org/protein/mcherry/)           | PA14: 160. LESB58: 63, 64, 131, 132. |

**NOTE: the strain LESB58 needs 500 ug/mL Gent, not 50ug/mL.**

[link-to-experimental-setup]

*Clearing up the [last review](https://pubmed.ncbi.nlm.nih.gov/36514980/)*

A brief recap of what I got up to yesterday. The current section talks about physical interactions. Cells are only free to expand when they have room to expand into - their whole thesis is that cell density in the inoculum, while seeming unimportant, may limit cellular competitiveness, if focal cells do not have enough time to form an established colony.

For interdependent cells, instead a greater load of bacteria is important. Having lots of *P. stutzeri*, in an interdependent culture, helped create more intermixing patterns. An effect like this has been shown in *E. coli* to create ecological niches. A greater inoculum, perhaps could help with creating co-operativity. That's a neat question to ask - at which time does bacterial co-operation, or interspecies tolerance, mature? - though I'm not sure how I'd go about looking at 'ecological' development.

Spatial genetic drift. Since only a small number of cells are replicating, there is less competition to overcome, and only a slightly faster replicating strain, could be enough to create a monoclonal sector. The front of replication can change, as *Pa* can be around 15um, and *Ec* can be around 30um. It's a neat little measurement. It's also neat that such small fitness advantages, can only occur prior to sector border formation, or early in biofilm development. It reminds me a lot of how an early-game advantage say, in something like a 4X game, can lead to a late-game economic advantage. This means that primarily-homogenous cells, like in the inoculum's nucleus, will remain homogenous, while cells at the borders can make heterogenous sectors. I'm not sure how this would happen in a polymicrobial setting, but I guess I could measure fitness with something like the macro-population models.

What makes a sector boundary, can be pretty variable. They say it's due to species composition (rod bacteria, apparently, make more jagged edges?) , ECM composition, cell filamentation, and environmental conditions. Lots of good references. I'm not sure about whether this is a thing for morphology though.

Papers have looked at impossible conditions. For example, when nutrients are not limited, there is no segregation. Likewise, as the rate of nutrient diffusion decreases, segregation increases.

It's neat that while genetic drift, or other such diplomatic affairs, can induce segregation, that it can be overcome. Strong mutualism, or environmental heterogeneity, may force cells to remain homogenous, despite being biologically opposing.

Ecological dynamics. Nomads and residents. This is a classical ecological theory, that cells may avoid local competition, by moving to uncompetitive areas, in this case the colony border. This could happen by flagellar motility, faster growth-rate, biofilm production, or a parasitic strategy. This is something of a paradox. In monoculture, strains may grow slower and have lesser bactericidal capacity, but they can perform better in polymicrobial biofilms. They give *Raoultella planticola* as an example. Moreover these processes can dominate, if the sector border relationship is mutually-antagonistic, so sectors are forced to expand radially. There's a Pa model that looks at this!

There's a bunch of theoretical work, that supports cell length as a way to cells may gain access to the biofilm's edge. I'm not sure what they mean exactly, but I can return to the references later.

Big ol' section on syntrophy ('crossfeeding.') They call it only a mild mutualism, as a stronger case could be made for one species acting as a detoxifier. What they mean is that one bug makes something that the other bug can consume for sustenance. These can be bilateral relationships, or unilateral.

This means that producers, rather than consumers, will lead the edge of colony biofilms. This pattern is not the same as for cells at the biofilm's footprint, which are truly homogenous, whereas cells that can grow instead form a dendritic pattern. These associations, even while weaker than a true mutualism, can still increase biomass, at least more than a monomicrobial biofilm. I guess that these systems can be vulnerable to parasites ('cheaters,') but it looks like partial segregation, may be enough to keep these cheating cells out of areas of high nutrient density.

![fractal-mixing]

In terms of how producers/consumers expand, it's clear that in early development, the producers (an independent phenotype) will have more access to the biofilm periphery. As they need to get rid of their metabolic exports, more consumers, be they pushed up there by the biofilm or by their own growth will get out to the periphery, where they'll make monomicrobial enclaves. These are best seen late in biofilm development.

*Again and again - the [same review](https://pubmed.ncbi.nlm.nih.gov/36514980/) but reading the references*

| Visual characteristic                  | Bacteria                                               | Confocal?                                                                                                                                                                                                                                                                | Agar conditions                                                                                                                                                               | Year | Mechanism                                                                                                                                                                                                                                                                                                                                                                                             | Measurements                                                              | Comments                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
| -------------------------------------- | ------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Wrinkliness                            | *B. subtilis*                                          | Sytox Green for cell death in *Bs*, supplemented to media (0.5uM.) Razor-slices for cross-section. For inoculum top-up, use ultradense culture, on top of a fine paintbrush. Water control for paintbrush.                                                               | Inoculum: has fluorescent beads (fluorospheres). These show where wrinkles converge. Agar: 1.5%, 3mm thickness                                                                | 2012 | Genes involved in ECM production (*srfA, spo0A, sinI, tasA, epsH*)                                                                                                                                                                                                                                                                                                                                    | Spatial complexity (heterogeneity); perimiter-to-area-ratio; P^2 / 4*pi*A | Mention of cell-death-patterns (CDP). Local biochemical stress drives cell death. Stiffer wrinkles means more wrinkle-separation. Cell death drives convergence; measured temporally. Can 'top-up' inoculum, to change starting density.                                                                                                                                                                                                                             |
| Wrinkliness                            | *B. subtilis*                                          | *tapA* (TasA)-yfp reporter.                                                                                                                                                                                                                                              | Not given                                                                                                                                                                     | 2013 | O2, complemented with NO2, and by reducing iron, acting by a phosphorylation cascade (*kinA, kinB, kinC, kinD*) or the terminal cytochromes (*caa3, bc, aa3*.) They measure respiration using NADH/NADH+ ratios.                                                                                                                                                                                      | By-eye                                                                    | Notes that Pa makes biofilm in microxic conditions, linked to phenazines. Making lots of phenocopies, all acting on the respiratory cytochromes. Neat that some of the *kin* genes only act at the colony periphery.                                                                                                                                                                                                                                                 |
| Isogenic colony antagonism             | *Paenibacillus dendritiformis*                         | NA                                                                                                                                                                                                                                                                       | 1.5% agar, with 2g/L peptone, for the 'T' strain. 0.7% agar, with 2g/L peptone, for the 'C' strain. Humidity at 2%. Inoculum: uncertain.                                      | 2010 | Biofilm antagonism. A protein called subtilisin, which promotes colony expansion, triggers sibling-lethal-factor (Slf) production.                                                                                                                                                                                                                                                                    | By-eye and with some macroscopic measurements (mm).                       | They use a high-humidity dry. 12mL for *4 days*, even if at 50% humidity, feels excessive. A case of ecological fratricide, but triggered only be multi-colony proximity.                                                                                                                                                                                                                                                                                            |
| Demarcation region (colony antagonism) | *Myxococcus xanthus* : both toxin and nontoxic strains | NA                                                                                                                                                                                                                                                                       | Demarcation media (0.5% casitone, 0.1% glycine, 0.1% serine, 0.1% alanine, 2Â mM MgSO4, 1Â mM CaCl2, 1.0Â mM IPTG, and 1.0% agar). Sounds a lot like swarming media.             | 2017 | Experimentally controlled. One has an IPTG-inducible toxin (*sitA*).                                                                                                                                                                                                                                                                                                                                  | Colony position, relative to the demarcation line (measures dominance.)   | Note that Pa also has demarcation lines. Can happen by contact-dependent killing. Lots of maths that goes from a simple growth model to toxin-/nutrient-dependent competition.                                                                                                                                                                                                                                                                                       |
| Colony merging                         | *S. typhimurium*                                       | NA                                                                                                                                                                                                                                                                       | Low glucose (<0.1%) and low agar concentration (0.5-0.7%)                                                                                                                     | 2009 | No molecular detail. Instead nutrient diffusion, and edge velocity, are given as measurements for merging.                                                                                                                                                                                                                                                                                            | Not sure how this was done exactly. Lots of biophysics equations.         |                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
| Wrinkliness                            | *B. subtilis* & *P. agglomerans*                       | *Bs* *mTurq* & *Pa* *Ypet*. Cut out whole colony, using a sterile scalpel, to catch the side-view of the biofilm colony. Just put it on a glass microscope slide. I don't get how they got their confocal sections. They say they use both a coverslip and agar overlay? | Agar: MSgg medium, 1.5% agar. Inoculum: OD0.5, 1:1 ratio, but only 1uL total. Challenge: spots of antibiotics, onto biofilm directly. Gentamicin stress (10âÎ¼l of 500âÎ¼g/mL.) | 2019 | *Bs* *tasA*-dependent. If absent then *Pa* *amsC* can help in coculture. Both species help to make a hybrid biofilm environment. In this case *Pa* makes more of a difference, at least to colony wrinkliness. Looks like *Pa* lives in the centre of biofilm colonies. Single *Bs* cells which are motile by *flgM* deletion are more important for wrinkliness, than chained cells - *lytABCDF*-KO. | By-eye                                                                    | A paper on biofilm partitioning/heterogeneity, which helps with antibiotic resistance. Dual-mutant experiments, where both help make the biofilms, a cool way to get conditional mutants, where you can manufacture mutualism. They find that it's only found in the 'subtilis' subclade. Source of wrinkliness, say that the chained cells lay over the *Pa* base as if they were sheets rather than sand-particles, which would stack up into unstable structures. |


*Horrors from the past: TM15 & SAHA/DBO*
I like this joke a lot lol. I need a better font for presentations though. Curse of the Pharoah, y'know?

This was not something I wanted to hear about again - this is like reaalllly bad. My data for the EM/DBO was shit. Not even an exaggeration. No replicates, no wild-type strains. And I ran out of 1,4-Diamino-2-butanone ([DBO](https://www.sigmaaldrich.com/NZ/en/product/aldrich/199338?srsltid=AfmBOopWrnBmsfZrgL79eKvnZWxp_2GCgU-qBpH5JJln_r2TJZfxxcon)) too. I'll try and convince Daniel, maybe to order some more DBO. Here's the options Quartzy gives [1PlusChem](https://www.1pchem.com/prod/3660-09-1)/[CenMed](https://cenmed.com/1-4-diamino-2-butanone-dihydrochloride-c09-0863-950/).

For TM15, I got a summary from Sam. He said we'll need to do some energetics with the Ouroboros, as well as a compound rescreen, and some mutant validation. The mutant work will need lots of TM15 though. I found an extra vial of the stuff but am not sure if it'll be enough for much more than a precursory screen. Again we probs need more drugggs.

*Parts of a photograph. Ramblings.*

Oh boy aesthetic theory. My favourite. A touch too much for this project maybe but it'll be useful for literature interpretation, and for grouping standard measurements, in particular. I'll write my thoughts as a brief exercise.

Given that it looks like the lit has a trend towards hierarchical groupings, I'll write about my thoughts as if they were on a hierarchical scale. Simpler things are at one end of the scale and more complex perspectives, will be at the extremities (and can affect both super-simple/super-complex pictures.)

I stand by size and shape as the lowest-order characteristics. I think it's important that, while these are inter-related, that both can be theoretically distinct. In particular, I think about the study of optical illusions, and of topology. Instead, I care more about size and shape as a dichotomy. The 'what' question, applies here to how things may become rotund or filament, rather than the shape's distribution. It isn't such an easy thing to measure in reality, as this needs me to look at development, as in: how does a shape change? , rather than at static geometry. It's great for looking at metabolite concentrations.

The next one is a little tricky to pin down but I feel that 'curvature', as in the shape of a fish-bone, is a decent characteristic. The simplest way to look at this would be with a Fourier series, maybe to see how filamentous-biofilms bend. Somehow however, I get a feeling that there are better ways to do these analyses. The Fourier analysis, for example, would be weird to perform over a time-series experiment. It'd be kind of like comparing apples/oranges, if curvature changes. Maybe I could see which points make equidistant circles, if I were to treat the curve like a protractor. That only needs some rudimentary mathematics.

Pattern-forming is also pretty important, but this is kind of like trying to measure tessellation. It needs me to create an intercellular unit, say something like a 'spiral unit', and see how each 'spiral unit' interlocks. I guess that specifically intermixing, and maybe shape fractal dimension, maybe some other stuff too could be useful here - but really anything that doesn't become harder to do in the context of a time-series experiment. The rules behind pattern-forming don't really change over time and with maturation, as they care more about biological optimums. It's also good for establishing 'zones', as pattern-forming cares a lot of colony symmetry.

Image termination. This sort of thing is my grouping for terms that deal with how images develop borders. I really struggle with this kind of thing but maybe there'll be a chance for me to stretch my mind a little in this discipline. No examples yet.

And the last thing is image development. It's by far and away the most complex, but not an impossible analysis. The trick is to look at time as if it were a discreet variable. When do things become very different? - that's only part of the question as better is: What do things look like when they become very different? - and: What were things like before they became very different? It's kind of like how in a certain evolutionary perspective, things happen in big 'evolutionary jumps', rather than a bunch of small cumulative events. I feel that while always important, the developmental perspective *per se* is very low resolution. It's kind of just giving a name to phases that look different by-eye, kind of like in phenotypic phylogenies. The cool thing is that many phases can be seen simultaneously, and that cells can cycle between many over their ecological lifespan. I think that this effect is called 'ecological synchrony'.

And one more thing! The maths here has been done over the last 20 years, mostly. The trouble is getting stuff like fractals to occur in 3-dimensional space. Here's a neat example of a mandelbulb (a 3D fractal.) It's pretty tough to replicate, but not impossible. I wonder if there's a way to detect 3d fractals, at least in the context of a biofilm?

![mandelbulb]

Why don't I do every one of these analyses? There's no harm unless I try and make inter-analysis comparisons. That's when the theory can get very messy incredibly quickly. I guess I'll work towards getting everything finished.

*Messing with the inoculum*

Maybe we can represent cell deposition. Something like how sandpiles work in self-organized criticality. Slowly drop cells onto a growing agar biofilm. Can we kill a biofilm with too many cells?

*Culture parameters*

I could try different humidities. All I'd need is a box and some wet paper towels and maybe a hygrometer.

*More fun words*

- *In toto* - in its totality (Latin)
